examples/inline_osr.sou

```diff
@@ -0,0 +1,15 @@
+function main ()
```

Comment:
Pick your favourite reason :)

- Many of our interesting examples have a before/after
- Run once and add it to repo, instead of running it each time I want to see it
- Maybe I want to show someone on my phone
- But really, I'm lazy